src/main/groovy/edu/oregonstate/mist/mealcard/resources/MealPlanResource.groovy

```diff
@@ -0,0 +1,49 @@
+package edu.oregonstate.mist.mealcard.resources
+
+import com.codahale.metrics.annotation.Timed
+import edu.oregonstate.mist.api.Resource
+import edu.oregonstate.mist.api.jsonapi.ResourceObject
+import edu.oregonstate.mist.api.jsonapi.ResultObject
+import edu.oregonstate.mist.mealcard.db.MealPlanDAO
+
+import javax.ws.rs.GET
+import javax.ws.rs.Path
+import javax.ws.rs.PathParam
+import javax.ws.rs.Produces
+import javax.ws.rs.core.Context
+import javax.ws.rs.core.MediaType
+import javax.ws.rs.core.UriInfo
+import javax.ws.rs.core.Response
+
+@Path("diningbalances")
+@Produces(MediaType.APPLICATION_JSON)
+class MealPlanResource extends Resource {
+
+    private MealPlanDAO mealPlanDAO
+    private URI endpointUri
```

Comment:
This variable wasn't used. Can we use it to create a self link in the api response? [Example method](https://github.com/osu-mist/academic-disciplines-api/blob/f27a83666a6bdf0fdfbacb64acfbd8bde4bdcabf/src/main/groovy/edu/oregonstate/mist/academicdisciplines/resources/AcademicDisciplinesResource.groovy#L72-L77)

Example response:
```
{
  "links": {},
  "data": {
    "id": "12345",
    "type": "balances",
    "attributes": [],
    "links": {
      "self":"https://api.oregonstate.edu/v1/diningbalances/12345"
    }
  }
}
```